pkg/virt-launcher/virtwrap/api/defaults.go

```diff
@@ -4,7 +4,7 @@ const (
 	resolvConf        = "/etc/resolv.conf"
 	DefaultProtocol   = "TCP"
 	DefaultVMCIDR     = "10.0.2.0/24"
-	DefaultVMIpv6CIDR = "fd2e:f1fe:9490:a8ff::/120"
+	DefaultVMIpv6CIDR = "fd10:0:2::/120"
```

Comment:
if we're going to make a change like this, it needs to go in ASAP before the next release. Otherwise this is going to get weird during kubevirt updates.

for example, if a VMI starts with the old ipv6 IP and then we do a KubeVirt update, if/when the VMI is live migrated later on it's IP is going to change. The whole point of masquerade is to provide a stable ip. We'd be breaking that.

Reply:
@ormergi I'm afraid you made a trap with the refactoring of the code. Would you please open a PR where we just change the IP address, nothing else. That may get in faster (before release).

Reply:
Thanks, I see it now :+1: 
I removed the refactoring commits.

Reply:
@davidvossel Hey somehow this PR is not part of v0.29.0 release,
Did you mean that this should be part of v0.30.0 release?